From f696537bd396d62b87c3bc9532aa8db69e85cd5b Mon Sep 17 00:00:00 2001
From: Marek Vasut <marex@denx.de>
Date: Sat, 20 May 2023 16:01:13 +0200
Subject: [PATCH 138/232] bayer2rgb: Inline the j=0 value

The j variable is used as an iterator further down in this code, but
here it can be just inlined in the macro parameters to make the code
easier to read. This is done in preparation for further changes. No
functional change.

Part-of: <https://gitlab.freedesktop.org/gstreamer/gstreamer/-/merge_requests/4686>
---
 gst/bayer/gstbayer2rgb.c | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/gst/bayer/gstbayer2rgb.c b/gst/bayer/gstbayer2rgb.c
index c1892a2ee..07d3d2b3f 100644
--- a/gst/bayer/gstbayer2rgb.c
+++ b/gst/bayer/gstbayer2rgb.c
@@ -431,9 +431,8 @@ gst_bayer2rgb_process (GstBayer2RGB * bayer2rgb, uint8_t * dest,
 
   gst_bayer2rgb_split_and_upsample_horiz (LINE (3 * 2 + 0), LINE (3 * 2 + 1),
       src + 1 * src_stride, bayer2rgb->width);
-  j = 0;
-  gst_bayer2rgb_split_and_upsample_horiz (LINE (j * 2 + 0), LINE (j * 2 + 1),
-      src + j * src_stride, bayer2rgb->width);
+  gst_bayer2rgb_split_and_upsample_horiz (LINE (0 * 2 + 0), LINE (0 * 2 + 1),
+      src + 0 * src_stride, bayer2rgb->width);
 
   for (j = 0; j < bayer2rgb->height; j++) {
     if (j < bayer2rgb->height - 1) {
-- 
2.40.1

